rmgpy/rmg/main.py

```diff
@@ -99,6 +100,8 @@ class RMG(util.Subject):
     Attribute                           Description
     =================================== ================================================
     `input_file`                        The path to the input file
+    `profiler`                          A cProfile.Profile object for time profiling RMG
+    `stats_file`                        File path to store the profile stats, usually RMG.profile
```

Comment:
Do you think it's necessary to add this as an attribute? We don't allow user specification of the filename, so it seems like we should just construct it using the `output_directory` each time.

Reply:
Fair point, I only added `stats_file` as an attribute to pass it down to this module from rmg.py. I can redefine it though from knowledge of the output directory here though, I'll make this change